build_scripts/build_protobuf.sh

```diff
@@ -31,7 +31,7 @@ if [ "${CC_COMP}" != "gcc" ]; then
       ${HOST_ARCH_OPTION} \
       ${BUILD_ARCH_OPTION} \
       ${SYSROOT_ARG} \
-      --with-protoc=/usr/local/bin/protoc \
+      --with-protoc=${INSTALL_PREFIX}/bin/protoc \
```

Comment:
I think it won't work when building with JetPack. In such case the `INSTALL_PREFIX` is the directory with binaries for target architecture, while the `protoc` should be build for host platform.

Reply:
Fixed in 3030b7f3b127f281bca2bc91fb37207a4afcd2a6